includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -142,7 +198,7 @@ function( $block ) {
 	 *
 	 * @return string Lesson HTML
 	 */
-	private function get_lesson_block_html( $block ) {
+	public function render_lesson_block( $block ) {
```

Comment:
Are we just making these public for tests? If we are or are not intending them to be used outside of the class, maybe we can at least do `@access private`. They seem like they should be protected at least.

Reply:
Moved to `protected` as that works too, in d3aec36.